### ISSUE TITLE
ios app crashed after registration and login

### ISSUE THREAD

**Issue by j0k3r** [wallabag/ios-app#147]
_From @wirsel on May 30, 2018 10:2_

### Issue details

iOS app for wallabag crashes after registration (with client_id and client_secret) and login (user name and password) with no warning or error description.

### Environment
* I used a 14 day trail account on "https://app.wallabag.it/"
* iOS 11.3.1
* Safari browser on iOS (iPad)

### Steps to reproduce/test case
* Create trial account with username and password on "https://app.wallabag.it/"
* Create client to get client_id and client_secret by following instructions on https://app.wallabag.it/developer/client/create
* install wallabag iOS app (https://itunes.apple.com/de/app/wallabag-2-offiziell/id1170800946) from apple store
* start iOS app on iPad
* follow registration instructions, enter server url (https://app.wallabag.it), former created clien_id/client_secret/username/password - > crash

_Copied from original issue: wallabag/wallabag#3658_

**Comment by MacLemon** [comment]
Same behaviour here with a slightly different setup.

iOS 10.3.3 (14G60)
Self hosted Wallabag server version 2.3.2

Browser login on the Mac works fine, I know my Wallabag instance is working ok.

Relevant part of the crash log:
```
Process:             wallabag [1622]
Path:                /private/var/containers/Bundle/Application/9901A020-9EBE-4E79-8234-D63817E661EA/wallabag.app/wallabag
Identifier:          fr.district-web.wallabag
Version:             99 (2.3.2)
Code Type:           ARM-64 (Native)
Role:                Foreground
Parent Process:      launchd [1]
Coalition:           fr.district-web.wallabag [769]


Date/Time:           2018-06-01 16:38:41.6803 +0200
Launch Time:         2018-06-01 16:37:12.5544 +0200
OS Version:          iPhone OS 10.3.3 (14G60)
Report Version:      104

Exception Type:  EXC_BREAKPOINT (SIGTRAP)
Exception Codes: 0x0000000000000001, 0x00000001009c8da0
Termination Signal: Trace/BPT trap: 5
Termination Reason: Namespace SIGNAL, Code 0x5
Terminating Process: exc handler [0]
Triggered by Thread:  0

Filtered syslog:
None found

Thread 0 name:  Dispatch queue: com.apple.main-thread
Thread 0 Crashed:
0   libswiftCore.dylib            	0x00000001009c8da0 0x100818000 + 1772960
1   libswiftCore.dylib            	0x00000001009c8da0 0x100818000 + 1772960
2   libswiftCore.dylib            	0x000000010089cdec 0x100818000 + 544236
3   wallabag                      	0x000000010006f53c 0x100040000 + 193852
4   wallabag                      	0x000000010006fc18 0x100040000 + 195608
5   Alamofire                     	0x0000000100120890 0x1000e8000 + 231568
6   Alamofire                     	0x000000010011bb40 0x1000e8000 + 211776
7   Alamofire                     	0x00000001000fd188 0x1000e8000 + 86408
8   libdispatch.dylib             	0x000000018c08a9e0 0x18c089000 + 6624
9   libdispatch.dylib             	0x000000018c08a9a0 0x18c089000 + 6560
10  libdispatch.dylib             	0x000000018c08f5e8 0x18c089000 + 26088
11  CoreFoundation                	0x000000018d1810c8 0x18d0a6000 + 897224
12  CoreFoundation                	0x000000018d17ece4 0x18d0a6000 + 888036
13  CoreFoundation                	0x000000018d0aeda4 0x18d0a6000 + 36260
14  GraphicsServices              	0x000000018eb19074 0x18eb0d000 + 49268
15  UIKit                         	0x0000000193369c9c 0x1932f5000 + 478364
16  wallabag                      	0x00000001000461c0 0x100040000 + 25024
17  libdyld.dylib                 	0x000000018c0bd59c 0x18c0b9000 + 17820
```

**Comment by michaelx** [comment]
Same.

iOS 10.3.3 (14G60)
Selfhosted Wallabag 2.3.2 on Pi 3.

Web version works without problems, but the iOS app crashes. The share extensions works though regardless, and new items are correctly added to the Wallabag DB.

**Comment by crendl** [comment]
Same here.

iOS 10.2.1 (14D27)
wallabag 2.3.2

**Comment by bourvill** [comment]
I think the problem has been solved with the latest version (2.3.3).

If the problem persists open a new ticket

**Comment by MacLemon** [comment]
I can confirm that this problem has been solved. After having successfully logged in for the first time, the app though still crashes upon every launch. I'll file a new bug on that. Thanks for fixing that one.
